priv/repo/migrations/20251126230802_add_time_zone_to_hosts.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddTimeZoneToHosts do
```

Comment:
nitpick:
I would rename this to end with `ToHostReadModel` and the same with the file name. The same way we did the with the last boot timestamp